### PR TITLE
Remove BOM support

### PR DESCRIPTION
**Pull request by yaelharel**
Fixes: #239

Signed-off-by: Yael Harel <yharel@vmware.com>

### CODE REVIEW

File: pkg/commands/build/status.go [hunk 1]
```diff
@@ -75,17 +67,12 @@ Therefore, you must have credentials to access the registry on your machine when
...
-					return displayBuildStatus(cmd, bld)
-				}
+				return displayBuildStatus(cmd, bld)
 			}
 		},
 	}
 	cmd.Flags().StringVarP(&namespace, "namespace", "n", "", "kubernetes namespace")
 	cmd.Flags().StringVarP(&buildNumber, "build", "b", "", "build number")
-	cmd.Flags().BoolVar(&bom, "bom", false, "only print the built image bill of materials")
 	commands.SetTLSFlags(cmd, &tlsConfig)
```
Comment (tylerphelan) [inline pkg/commands/build/status.go:76]: these flags can be removed too! no more tls needed!

Comment (yaelharel) [reply pkg/commands/build/status.go]: Thanks for your comment! Please see https://github.com/vmware-tanzu/kpack-cli/pull/261/commits/7d5a6a995a31b5d41fb264c0a9ef39c831b40540.

**Comment by tomkennedy513** [comment]
maybe we can squash these commits together

File: pkg/commands/build/status.go [hunk 2]
```diff
@@ -27,15 +26,12 @@ import (
 
 const (
 	buildMetadataKey = "io.buildpacks.build.metadata"
-	bomKey           = "bom"
 )
 
 func NewStatusCommand(clientSetProvider k8s.ClientSetProvider, rup registry.UtilProvider) *cobra.Command {
```
Comment (tylerphelan) [inline pkg/commands/build/status.go:31]: oh wait I missed this before. The `registry.UtilProvider` can be removed as well

File: pkg/commands/build/status_test.go [hunk 3]
```diff
@@ -77,7 +77,6 @@ bp-id-2         bp-version-2         mysupercoolsite2.com
 		clientSetProvider := testhelpers.GetFakeKpackProvider(clientSet, defaultNamespace)
 
 		fakeFetcher := registryfakes.Fetcher{}
```
Comment (tylerphelan) [inline pkg/commands/build/status_test.go:79]: the fake fetcher and other objects related to registry access can be removed in the test

File: pkg/commands/build/status.go [hunk 4]
```diff
@@ -27,15 +26,12 @@ import (
 
 const (
 	buildMetadataKey = "io.buildpacks.build.metadata"
-	bomKey           = "bom"
```
Comment (matthewmcnew) [inline pkg/commands/build/status.go:30]: Is buildMetadataKey used?

**Review by matthewmcnew — APPROVED** [review]

File: pkg/commands/build/status.go (hunk 4 above)
Comment (yaelharel) [reply pkg/commands/build/status.go:30]: No, I removed it as part of 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.

File: pkg/commands/build/status_test.go (hunk 3 above)
Comment (yaelharel) [reply pkg/commands/build/status_test.go]: Please see 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.

File: pkg/commands/build/status.go (hunk 2 above)
Comment (yaelharel) [reply pkg/commands/build/status.go]: Thanks! Please see 831e0496cbc9a6ee8da4e5a997a5487d6c0a8714.

**Review by tylerphelan — APPROVED** [review]
Nice!!

**Review by tomkennedy513 — APPROVED** [review]

**Pull request merged**